bot/cogs/utils.py

```diff
@@ -250,8 +191,46 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
         for reaction in options:
             await message.add_reaction(reaction)
 
-    async def send_pep_zero(self, ctx: Context) -> None:
-        """Send information about PEP 0."""
+    # PEPs area
+
+    @loop(hours=3)
+    async def refresh_peps_urls(self) -> None:
```

Comment:
Instead of polling, how about trying to get from the existing cache first and then sending a request if a PEP doesn't exist? A more complex variation of this idea is to store PEPs that couldn't be found in a separate cache which gets cleared once in a while. This would avoid spamming HTTP requests if the same bad PEP is requested within a short time frame, but I don't really think such complexity is necessary.